sn_node/src/node.rs

```diff
@@ -410,6 +410,39 @@ impl Node {
                     quotes_verification(&network, quotes).await;
                 });
             }
+            NetworkEvent::ChunkProofVerification {
+                peer_id,
+                keys_to_verify,
+            } => {
+                event_header = "ChunkProofVerification";
+                let network = self.network.clone();
+
+                trace!("Going to verify chunk {keys_to_verify:?} against peer {peer_id:?}");
+
+                let _handle = spawn(async move {
+                    // To avoid the peer is in the process of getting the copy via replication,
+                    // repeat the verification for couple of times (in case of error).
+                    // Only report the node as bad when ALL the verification attempts failed.
+                    let mut attempts = 0;
+                    while attempts < 3 {
+                        if chunk_proof_verify_peer(&network, peer_id, &keys_to_verify).await {
+                            return;
+                        }
+                        // Replication interval is 22s - 45s.
+                        // Three times 15 gives 45s, which ensures at least one replication
+                        // shall be carried out during the period.
+                        tokio::time::sleep(std::time::Duration::from_secs(15)).await;
```

Comment:
can we clean this up and get the time in a documented const, please 🙇 (same with `attempts`)